src/ammer/conv/Bytes.lua.hx

```diff
@@ -13,13 +12,8 @@ abstract Bytes(HaxeBytes) from HaxeBytes to HaxeBytes {
     return @:privateAccess new HaxeBytes(ptr.length, data);
   }
 
-  public function toNative1():String {
-    var b = this.getData();
-    var t:Table<Int, String> = Table.create();
-    for (i in 0...b.length) {
-      Table.insert(t, NativeStringTools.char(b[i]));
-    }
-    return Table.concat(t);
+  public function toNative1(): HaxeBytes {
+    return this;
```

Comment:
I wasn't really sure what was going on here... the rest of the targets were returning bytes, but this was set up to create a string, and it was failing.

Reply:
This is kind of ugly because `haxe.io.BytesData` on Lua is an alias for `Array<Int>`. It really should be a `String`, since Lua strings are binary-safe …

Reply:
Yeah, this is sort of a "chicken and egg" problem.  I actually want to use Ammer to create a simple bytes/buffer type in Haxe.  I might need something separate for that.

Reply:
Why not just use the native `String` as `BytesData`? Are Lua strings immutable or something?